setup.py

```diff
@@ -109,7 +109,7 @@ def find_version(*file_paths):
     package_data={
         'cylc.flow': [
             'etc/*.yaml', 'etc/flow*.eg', 'etc/job.sh',
-            'etc/syntax/*', 'etc/cylc-bash-completion'
+            'etc/syntax/*', 'etc/cylc-bash-completion', 'py.typed'
```

Comment:
Just to check my understanding this is an empty file that gets bundled with the package (that never gets added to?) that acts as a flagfile to mypy?

Reply:
Yes, I believe so 🤷